js/src/chrome/settings/chrome.settings.extras.js

```diff
@@ -439,8 +439,13 @@ Wu.Chrome.SettingsContent.Extras = Wu.Chrome.SettingsContent.extend({
 		console.log('layerJSON', layerJSON);
 
 		// create layer on server
-		Wu.post('/api/db/createLayer', JSON.stringify(layerJSON), function (err, newLayerJSON) {
-
+		app.api.createLayer(layerJSON, function (err, newLayerJSON) {
+			if (err) {
+				return app.feedback.setError({
+					title : 'Something went wrong in _updateLayer',
+					description : err
+				});
+			}
 			// new layer
 			var newLayerStyle = Wu.parse(newLayerJSON);
 
```

Comment:
done() below in line 454 should probably return done(err) if there is an error....